applications/RANSApplication/python_scripts/rans_analysis.py

```diff
@@ -0,0 +1,59 @@
+from sys import argv
+
+import KratosMultiphysics as Kratos
+
+from KratosMultiphysics.analysis_stage import AnalysisStage
+from KratosMultiphysics.RANSApplication.coupled_rans_solver import CoupledRANSSolver
+
+class RANSAnalysis(AnalysisStage):
+    def __init__(self, model,parameters):
+        """Main script for RANS fluid dynamics simulations using formulations via coupled_rans_solver
+
+        Args:
+            model (Kratos.Model): Model used in the simulation
+            parameters (Kratos.Parameters): Parameters used in the simulation
+        """
+        super().__init__(model, parameters)
+
+    def _CreateSolver(self):
+        return CoupledRANSSolver(self.model, self.project_parameters["solver_settings"])
+
+    def _GetOrderOfProcessesInitialization(self):
+        # The list of processes will contain a list with each individual process already constructed (boundary conditions, initial conditions and gravity)
+        # Note 1: gravity is constructed first. Outlet process might need its information.
+        # Note 2: initial conditions are constructed before BCs. Otherwise, they may overwrite the BCs information.
+        return ["gravity",
+                "initial_conditions_process_list",
+                "boundary_conditions_process_list",
+                "auxiliar_process_list"]
```

Comment:
I'd get this from the base `FluidDynamicsAnalysis`.

Reply:
Done locally, will push once you are done with the review :)